.github/workflows/iroha2-dev-pr-wasm.yaml

```diff
@@ -40,7 +47,43 @@ jobs:
         if: always()
         run: cargo doc --no-deps --quiet
 
-  tests:
+  wasm-validator-static-analysis:
```

Comment:
do we have to have two steps for wasm static analysis?

Reply:
I belive, that no, thanks to your suggestion about workspace